code/__DEFINES/misc.dm

```diff
@@ -38,6 +38,7 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 #define VALENTINES				"Valentine's Day"
```

Comment:
No, that's not what this is for, it's for checking against holiday strings, not for just listing all holidays.
Technically I shouldn't have added the ```SPACE_EASTER``` one but I figured someone might want to make Space Easter slightly different to Easter at some point, so I pre-emptively added it.